ups/eupspkg.cfg.sh

```diff
@@ -17,11 +17,11 @@ install()
     case $(uname -s) in
         Linux*)
             ana_platform="Linux-x86_64"
-            conda_packages="conda_packages-linux-64.txt"
+            conda_packages="conda2_packages-linux-64.txt"
             ;;
         Darwin*)
             ana_platform="MacOSX-x86_64"
-            conda_packages="conda_packages-osx-64.txt"
+            conda_packages="conda2_packages-osx-64.txt"
```

Comment:
And sure, it might break a bit later on when it tries to install packages that are no longer valid for the miniconda2 that is being installed but that's a failure that this change doesn't actually solve.